CoreUtilities/test/unit/src/au/gov/asd/tac/constellation/utilities/json/JsonUtilitiesNGTest.java

```diff
@@ -67,6 +78,540 @@ public void setUpMethod() throws Exception {
     public void tearDownMethod() throws Exception {
     }
 
+    
+    /**
+     * Test that the default (private) constructor throws an exception and does nothing.
+     * @throws Exception 
+     */
+    @Test
+    public void testDefaultconstructor() throws Exception {
+        
+        Constructor<JsonUtilities> constructor = JsonUtilities.class.getDeclaredConstructor();
+        constructor.setAccessible(true); 
+        try {
+            constructor.newInstance();
+        } catch (InvocationTargetException  e) {
+            // Found I couldnt add @Test(expectedExceptions=BLAH) doe to how reflection wrpas exceptions.
+            assertEquals(e.getCause().toString(), "java.lang.IllegalStateException: Utility class", "Correct exception");
+        }
+    }
+    
+    /**
+     * Test calls to JsonUtilities.getTextField for which no default is supplied.
+     */
+    @Test
+    public void testGetTextField_NoDefault() {
+        try {
+            ObjectMapper mapper = new ObjectMapper();   
+            JsonNode testJson = mapper.readTree("{\"1.k1\":\"1.v1\", \"1.k2\":\"1.v2\", \"1.k3\":{\"2.k1\": \"2.v1\"}}");  
+            
+            // Search for missing top level value
+            assertEquals(JsonUtilities.getTextField(testJson, "1.Missing"), null, "L1 null returned if not found");
+            
+            // Search for existing top level value
+            assertEquals(JsonUtilities.getTextField(testJson, "1.k2"), "1.v2", "L1 value returned if found");
+            
+            // Search for missing nested value
+            assertEquals(JsonUtilities.getTextField(testJson, "1.k3", "2.Missing"), null, "L2 null returned if not found");
+            
+            // Search for existing nested value
+            assertEquals(JsonUtilities.getTextField(testJson, "1.k3", "2.k1"), "2.v1", "L2 value returned if found");  
+        } catch (JsonProcessingException e) {
+            fail("Exception thrown generating JSON for test");
+        }  
```

Comment:
I think I would just make the test throw the exception rather than catch it here. Similarly for subsequent tests

Reply:
done